docs/api.rst

```diff
@@ -1372,6 +1372,18 @@ If you just want to filter the films by actors but don't want to include them in
     }
   ]
 
+The top level resource can also be sorted by the embedded resource columns in :ref:`Many-to-One <many-to-one>` and :ref:`One-to-One <one-to-one>` relationships. For example, sorting the films by the director's last name in descending order:
+
+.. tabs::
+
+  .. code-tab:: http
+
+    GET /films?select=title,directors(last_name)&order=directors(last_name).desc HTTP/1.1
+
+  .. code-tab:: bash Curl
+
+    curl "http://localhost:3000/films?select=title,directors(last_name)&order=directors(last_name).desc"
+
```

Comment:
This feature gets a bit lost with the current structure.

I think it should be below `Embedding with Top-level Filtering` and under the `"Top-Level Ordering"` subheading.

This way the feature can also be linked in the release notes.

Reply:
"related orders" as the name for the feature will make more sense when we add "related filters".